x/authority/keeper/keeper_test.go

```diff
@@ -118,6 +129,58 @@ func TestCreateAndRevokeIssuer(t *testing.T) {
 	require.Empty(t, ik.GetIssuers(ctx))
 }
 
+func TestReplaceAuthUseBothAuthorities(t *testing.T) {
+	ctx, keeper, ik, _ := createTestComponents(t)
+
+	var (
+		accAuthority    = mustParseAddress("emoney1kt0vh0ttget0xx77g6d3ttnvq2lnxx6vp3uyl0")
+		accNewAuthority = mustParseAddress("emoney17up20gamd0vh6g9ne0uh67hx8xhyfrv2lyazgu")
+		issuer1         = mustParseAddress("emoney17up20gamd0vh6g9ne0uh67hx8xhyfrv2lyazgu")
```

Comment:
No, because we do not place such restrictions: preventing the authority to become an issuer. But being counter-intuitive is a good justification to replace it.